contracts/common/Errors.sol

```diff
@@ -9,3 +9,4 @@ error ZeroAddress();
 error SendFailed();
 error InvalidCommission(uint256 commission);
 error InvalidMinStake(uint256 minStake);
+error InvalidMinDelegation(uint256 minDelegation);
```

Comment:
Use this file for common errors only. All other errors are to be declared in the module that uses it. It is better not to use error parameters so less gas is spent. The minDelegation value can be seen in the tx data when needed.

Reply:
removed uint256 from minStake and minDelegation!